src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -16,6 +16,17 @@ public SelectTestFigureOptionListener(DriverManager driverManager) {
 
 	@Override
 	public void actionPerformed(ActionEvent e) {
-		FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+		String value = e.getActionCommand();
+
+		switch (value){
+			case "Figure Joe 1":
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.